sos/report/plugins/kernel.py

```diff
@@ -147,4 +148,12 @@ def setup(self):
         if not self.get_option("trace"):
             self.add_forbidden_path("/sys/kernel/debug/tracing/trace")
 
+
+class AzureKernel(Kernel, AzurePlugin):
```

Comment:
The way the tagging resolution works out, we'd have to change `IndependentPlugin` to the specific `RedHatPlugin`, `UbuntuPlugin`, etc... tags if we break out one for just Azure Linux here.

That said......how have we not been collecting this generally for so long? @pmoravec @arif-ali should this become a general collection? I'm surprised that we don't do this today already, but I have to assume that no one has been asking for it until now...somehow?

I think we'd probably want this to go into the regular collection, and keep `Kernel` as a strictly `IndependentPlugin`.

Reply:
Ah I see, the `IndependentPlugin` are for plugins that are distribution agnostic?

Happy to move this to the regular collection (the kernel setup function) above and keep `Kernel` as a strictly `IndependentPlugin`

Reply:
Agreed, that could be useful for Ubuntu Plugin as well, so agree to add this above in the main class

Reply:
@TurboTurtle @arif-ali, I added `/boot/config-{kernel-version}` to the main class above and removed the AzureKernel specific class